ci-infra/azure/main.tf

```diff
@@ -30,7 +30,7 @@ resource "azurerm_federated_identity_credential" "gh_action_federated_credential
   parent_id           = azurerm_user_assigned_identity.gh_action_user_identity.id
 
   # TODO: What should be the subject when we are not testing just main but the respective PR?
-  subject = "repo:${var.gh_repo}:pull_request"
+  subject = "repo:${var.gh_repo}:ref:refs/heads/main"
```

Comment:
Is there a way to add both subjects? So we support running on main and running-on-PR-manually?

Reply:
I don't think you can use workflow-dispatch on PRs, in the drop-down you can choose a branch for the main repo, maybe the subject accepts wildcards like `refs/heads/*`, but that would only work for branches on the main repo, not forks, i.e. not useful for PRs

Reply:
Alright ok.